resources/metadata/helmExecute.yaml

```diff
@@ -0,0 +1,371 @@
+metadata:
+  name: helmExecute
+  aliases:
+    - name: deployToKubernetes
+      deprecated: true
+  description: Deployment to Kubernetes test or production namespace within the specified Kubernetes cluster.
```

Comment:
description should rather be about helm execution since this goes beyond pure deployment

Reply:
changed description